src/main/java/frc/robot/subsystems/claw/ClosedCone.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.claw;
+
+import frc.robot.RobotMap;
+import frc.robot.lib.statemachine.State;
+import frc.robot.lib.statemachine.Transition;
+
+/** Add your docs here. */
+public class ClosedCone extends State {
+    
+    @Override
+    public void build() {
+        // open claw if open request
+        transitions.add(new Transition(() -> {
```

Comment:
We should also add a transition if we don't detect a cone, go to the next state. Update state machine [diagram](https://lucid.app/lucidchart/b6090843-0b3b-4749-8b86-71403e67fac1/edit?invitationId=inv_7482c25f-2845-4fb0-9f3d-8a34f9c4208e&referringApp=slack&page=0_0#) to reflect this.